NicolaeChristof/Rise
# Feature uibackdrop

In this branch I added an outline to most of the text (except the loading screen, since that already has a green backdrop). This is to make the text stand out a bit more. Additionally, I added a leaf to the back of the timer UI element as a test. I wasn't sure if I did this correctly design-wise, so let me know whether you want it in or not.

How to Test:
Make sure that all the text (except the loading screen) is outlined and let me know whether you like the leaf behind the timer.

Known Glitches:
The leaf behind the timer stays when you enter the pause menu.
